src/neptune/internal/operation_processors/operation_storage.py

```diff
@@ -27,6 +25,14 @@
 from neptune.internal.utils.logger import logger
 
 
+def get_container_dir(type_dir: str, container_id: UniqueId, container_type: ContainerType) -> str:
```

Comment:
This func is basically the clue of this PR. The rest is just adjusting the existing code for its usage